forcats.Rmd

```diff
@@ -296,17 +296,17 @@ countries_in_1896 %>%
   levels()  
 ```
 ### Reordering levels by other variables
-The functions presented in this section bare great similarity to the `fct_relevel()` function introduced in the beginning of this section.
-`fct_relevel()` allows for a direct manipulation of the levels by passing the new order to the `levels` keyword argument. In this regard `fct_reorder()` and `fct_reorder2()` are different. Here the levels are reordered according to the result of a function applied to one vector `x` in case of `fct_reorder()` and two vectors `x` and `y` in case of `fct_reorder2()`.
+The functions presented in this section bare great similarity to the `fct_relevel()` function introduced in the beginning of this chapter.
+`fct_relevel()` allows a direct manipulation of the levels by passing the new order to the `levels` keyword argument. At this point `fct_reorder()` and `fct_reorder2()` are different. Here, the levels are reordered according to the result of a function applied to one vector `x` in case of `fct_reorder()` and two vectors `x` and `y` in case of `fct_reorder2()`.
```

Comment:
To me, **At this point...** also has some temporal component. 
This is not what we mean here.
Thus, I would rather stick with the original form.

Reply:
I would also suggest the original form